inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -342,30 +342,11 @@ persistent_volumes_enabled: false
 ## Support tls min version, Possible values: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
 # tls_min_version: ""
 
-## Support tls cipher suites.
+## Support tls cipher suites. See https://pkg.go.dev/crypto/tls for more cipher suites.
```

Comment:
HI @yckaolalala 

If changing the `https://pkg.go.dev/crypto/tls` to `https://pkg.go.dev/crypto/tls#pkg-constants` , it can make it clear :-)

Reply:
@yankay done